src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -495,6 +528,9 @@
     text: az acr repository show-tags -n MyRegistry --repository MyRepository --detail
 -   name: Show the detailed information of the latest 10 tags ordered by timestamp of a repository in an Azure Container Registry.
     text: az acr repository show-tags -n MyRegistry --repository MyRepository --top 10 --orderby time_desc --detail
+-   name: Show tags for a repository in an Azure Container Registry.
+    text: az acr repository show-tags --output json --repository MyRepository --name MyRegistry
```

Comment:
Less detailed than existing example. Generally speaking, adding `-o json` to an example doesn't add value because that's the default output format anyways.